website/versioned_docs/version-5.x/ecosystem-angular.md

```diff
@@ -88,6 +88,15 @@ work with Angular 9. Follow the [Angular CLI instructions](#angular-cli).
 Note that the schematics for Angular 9 [do not use the custom Angular builder](#angular-builder), but instead use
 [@angular-builders/custom-webpack](https://www.npmjs.com/package/@angular-builders/custom-webpack).
 
+### Angular 10
+Angular 10 is supported by single-spa-angular@4.4.0+.
```

Comment:
I'm worried that newly published versions of single-spa-angular 4 will not work with Angular 9 projects since the single-spa-angular dependencies are Angular 10 now. Have you been able to confirm whether the code on the master branch of single-spa-angular works with Angular 9 still? If not, we will have to create a new major version

Reply:
I will check on weekends.

Reply:
@joeldenning I have downgraded Angular to `9.0.0` version in `single-spa/integration/chat` (where we have integration apps) and checked with the latest single-spa-angular version, it works. I removed `"4.4.0+"`.